samples/app-complete-auth/csharp/AppCompleteAuth/AppManifest/manifest.json

```diff
@@ -109,7 +130,8 @@
           "type": "link",
           "value": {
             "domains": [
-              "*.botframework.com"
+              "*.botframework.com",
+              "<<base-url>>"
```

Comment:
What will be the base-url value?
Did we mention in the Readme file?

Reply:
Yes it is mentioned.